compiler/rustc_target/src/spec/armv7_unknown_trusty.rs

```diff
@@ -0,0 +1,33 @@
+use crate::spec::{Target, TargetOptions};
+
+use super::{crt_objects::LinkSelfContainedDefault, PanicStrategy, RelroLevel};
+
+pub fn target() -> Target {
+    Target {
+        // It's important we use "gnueabi" and not "musleabi" here. LLVM uses it
+        // to determine the calling convention and float ABI, and it doesn't
+        // support the "musleabi" value.
+        llvm_target: "armv7-unknown-unknown-gnueabi".into(),
```

Comment:
Really? `unknown-unknown` instead of e.g. `unknown-linux`? That makes sense to LLVM? Huh. Learn something new every day.